codespell_lib/data/dictionary_rare.txt

```diff
@@ -75,6 +75,8 @@ girds->grids
 guarantied->guaranteed
 guerilla->guerrilla
 guerillas->guerrillas
+habitant->inhabitant
+habitants->inhabitants
```

Comment:
what about `habitat(s)` ?

Reply:
Is `habitant` that rare? I understand native French speakers tend to prefer `habitant` over `inhabitant`, but `habitant` remains an English words as far as I can see:
* Cambridge: [habitant](https://dictionary.cambridge.org/dictionary/english/habitant)
* Oxford: -
* Collins: [habitant](https://www.collinsdictionary.com/dictionary/english/habitant): _a less common word for [inhabitant](https://www.collinsdictionary.com/dictionary/english/inhabitant)_
* Merriam-Webster: [habitant](https://www.merriam-webster.com/dictionary/habitant): _a settler or descendant of a settler of French origin working as a farmer in Canada_

Also, I think `habitat` is a real word in science, not even rare:
* Cambridge: [habitat](https://dictionary.cambridge.org/dictionary/english/habitat)
* Oxford: [habitat](https://dictionary.cambridge.org/dictionary/english/habitat)
* Collins: [habitat](https://www.collinsdictionary.com/dictionary/english/habitat)
* Merriam-Webster: [habitat](https://www.merriam-webster.com/dictionary/habitat)